spec/index.bs

```diff
@@ -11681,6 +11680,7 @@ GPUQueue includes GPUObjectBase;
                         - |source|.|origin|.[=Origin3D/z=] + |copySize|.[=Extent3D/depthOrArrayLayers=]
                             must be &le; 1.
                     </div>
+                1. Let |usability| be [=?=] [=check the usability of the image argument=](|source|).
```

Comment:
Technically this returns a value "synchronously" within the JS call to this function, but the result is not actually used synchronously, so under "[as-if](https://gpuweb.github.io/gpuweb/#w3c-conformant-algorithms)" I think it's fine.